src/abi/global.ts

```diff
@@ -214,32 +215,40 @@ export const GlobalFunctions: Map<string, AbiFunction> = new Map([
                     return `${ctx.used("__tact_nop")}()`;
                 }
                 const arg = args[0];
+
+                const filePath = ref.file
+                    ? path.basename(ref.file!)
+                    : "unknown";
+                const lineCol = ref.interval.getLineAndColumn();
+                const debugPrint = `[DEBUG] File ${filePath}, Line ${lineCol.lineNum}, Column ${lineCol.colNum}`;
```

Comment:
let's do it in the standard way: `filename:line:column` -- most text editors can then open the file at this exact line and column when you click on this in your terminal, for instance